run_cmake_build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Configure cmake to build torch-mlir in-tree
+cmake -GNinja -Bbuild \
```

Comment:
This are normal cmake commands so we probably want to avoid this since everyone has a variation of these commands to run.